README.md

```diff
@@ -16,6 +16,8 @@ It requires:
 
```

Comment:
### **Hi @itmilos,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- You should explain we use bundle install, yarn, bin/setup, and bin/dev for trigger foreman and the app is visible on localhost:3000 in the browser
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏